build.sbt

```diff
@@ -1,6 +1,6 @@
 name := "tempest"
 
-version := "0.12.0"
+version := "0.13.0"
```

Comment:
Here we need to declare a version so that users can build against past versions if needed. I guess until the 1.0 release we could use a version like `0.13.0-SNAPSHOT`; I think for `SNAPSHOT` versions, users understand the code might change out from under them, and the central repo allows the version to be redefined.  But I'm not sure I like that, and I think some of Teapot's code might actually build against a previous version of Tempest.